modules/form/README.md

```diff
@@ -0,0 +1 @@
+# WSO2 Identity Server - Form Module for Javascript/Typescript Applications
```

Comment:
Shall we say `React Applications`, since the module is based on React.
```suggestion
# WSO2 Identity Server - Form Module for React Applications
```